projects/plugins/jetpack/extensions/blocks/google-docs-embed/editor.scss

```diff
@@ -0,0 +1,99 @@
+/**
+ * Editor styles for Google Docs
+ */
+ @import './view.scss';
+
+.wp-block-jetpack-google-docs-embed {
+	margin: 0 0 1.5em 0;
+	clear: both;
+
+	&__wrapper {
+		position: relative;
+
+		&::before {
+			content: "";
+			display: block;
+			padding-top: 50%;
+		}
+	}
+
+	&__interactive-overlay {
+		position: absolute;
+		top: 0;
+		left: 0;
+		right: 0;
+		bottom: 0;
+		opacity: 0;
+	}
+
+	&-embed-form-editor {
+		margin: 0;
+
+		textarea {
+			margin-right: 1px;
+			flex: 1;
+			height: 36px;
+			padding-top: 9px;
+			font-family: inherit;
+			font-size: inherit;
+		}
+	}
+
+	iframe {
+		max-width: 100%;
+		position: absolute;
+		top: 0;
+		right: 0;
+		bottom: 0;
+		left: 0;
+		height: 100%;
+		width: 100%;
+	}
+
+	.wp-block[data-align="left"] > &,
+	.wp-block[data-align="right"] > &,
+	&.alignleft,
+	&.alignright {
+		max-width: 360px;
+		width: 100%;
+
+		// Unless these have a min-width, they collapse when floated.
+		&__wrapper {
+			min-width: $break-zoomed-in;
+		}
+	}
+
+	&-editmode {
+		.logo {
+			width: 24px;
+			height: 24px;
+			margin-right: 10px;
+		}
+
+		.components-base-control {
+			width: 100%;
+		}
+
+		.components-base-control__label {
+			text-align: left;
+		}
+
+		.components-base-control__help {
+			color: #ff4500;
+		}
+
+		iframe {
+			pointer-events: none;
+		}
+
+		&-sidebar {
+			margin-top: 16px;
+
+			hr {
+				margin: 16px -16px;
+				border: none;
+				border-top: 1px solid #e2e4e7;
```

Comment:
I would recommend pulling colors from WordPress' base styles package, to ensure the colors remain consistent with Core across updates. You can find some examples of this across the repo, and see this for  more info:
https://github.com/Automattic/jetpack/blob/d2d62ec3c350533ae2629e0fec60aa56fc83cc75/projects/plugins/jetpack/extensions/README.md#L339-L343

Reply:
Noted. This section was in fact a carry over and I've removed it entirely.